devices/generic/items/state_expectedeventduration_item.json

```diff
@@ -0,0 +1,8 @@
+{
+	"schema": "resourceitem1.schema.json",
+	"id": "state/expectedeventduration",
+	"datatype": "UInt16",
+	"access": "R",
+	"public": true,
+	"description": "The expected duration of the last received rotary event."
```

Comment:
Might be beneficial to include also `"default": 0` for all the to be added resource items

Reply:
Wouldn't it default to `null`?

Reply:
Yes. It's just beauty I suppose...